index.js

```diff
@@ -3778,6 +3778,24 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# fromPairs :: Array (Pair String a) -> StrMap a
+  //.
+  //. Returns a new string map formed by a list of key-value pairs.
+  //. If a key appears in multiple pairs, the rightmost pair takes precedence.
+  //.
+  //. ```javascript
+  //. > S.fromPairs([['a', 1], ['b', 2], ['c', 3]])
+  //. {b: 2, a: 1, c: 3}
```

Comment:
Is there a reason to write `{b: 2, a: 1, c: 3}` rather than `{a: 1, b: 2, c: 3}`?

Reply:
Copy-pasted from `S.pairs`. I guess it's a hint on the arbitrary order of things? Should we: leave it as is, sort only `fromPairs`, or sort both `pairs` and `fromPairs` examples?

Reply:
Oh, wait, I see that `S.pairs` example is sorted explicitly with `.sort()`. I'll just sort the keys in `fromPairs` example then.

Reply:
:thumbsup: